README.md

```diff
@@ -62,6 +62,18 @@ So in a given Zome we have two things:
 
 In order for Holochain to run your app, you have to build your code into a single packaged file. Those instructions follow.
 
+## Ignoring files
+
+Sometimes, you'll want to exclude files in your project directory to get a straight .hcpkg file that can be understood by Holochain. In order to do that, just create a `.hcdevignore` file. It has a similar structure to `.gitignore` files:
```

Comment:
at the risk of being "that guy" and bike shedding this to death... what do you think of just `.hcignore`?

Reply:
Also the tool also recognizes `.ignore` files, so we could do just tha

Reply:
Lets use `.hcignore` rather than `.hcdevignore` because we know the command name will change.